.travis.yml

```diff
@@ -0,0 +1,98 @@
+# Travis CI configuration file.
+# @link https://travis-ci.org/
+
+# For use with the Twenty Nineteen WordPress theme
+# @link https://github.com/WordPress/twentynineteen/
+
+# Declare project language and PHP versions to test against.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Newer versions like trusty don't have PHP 5.2 or 5.3
+# https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming
+dist: precise
+
+# Declare versions of PHP to use. Use one decimal max.
+php:
+    - "7.2"
```

Comment:
PHP 7.3 can be included as well, although it should be allowed to fail, since it's not released yet.

[Reference on WPCS repo](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1503/files)